Gemfile.lock

```diff
@@ -74,12 +74,12 @@ GEM
     factory_girl_rails (4.7.0)
       factory_girl (~> 4.7.0)
       railties (>= 3.0.0)
+    font-awesome-rails (4.6.3.1)
```

Comment:
Why is this changing here? It looks like one group might have a different Gemfile.lock than the other so each time one merges in, it means the other has a diff from master. Try to get everyone on your team using the same Gemfile.lock over the course of today. One suggestion of how to do this is by not committing changes to your Gemfile.lock on branches that do not need to actually change /update it